entry.js

```diff
@@ -15,7 +15,7 @@ Entry.prototype.forContents = function (cb) {
 }
 
 Entry.prototype.computePath = function (entitypath) {
-  this.path = path.relative('/', entitypath)
+  this.path = path.relative(path.sep, entitypath)
```

Comment:
This should be fine on windows. (It turns out that `/` is treated identically to `\\` in Windows APIs.)